anago

```diff
@@ -971,9 +971,32 @@ announce () {
     logecho "$WARNING: Email cannot be sent from GCB.  Execute the following" \
             "after a completed release to send email notification:"
     logecho
+
+    # for an explaination of this construct look at
+    #
+    # http://tldp.org/LDP/abs/html/testconstructs.html
+    #
+    # or for the tl;dr check out
+    #
+    # https://www.tldp.org/LDP/abs/html/dblparens.html
+    #
+    # what we're doing here is setting nomock_flag="--nomock"
+    # if FLAGS_nomock evaluates to true...it's concise and consistent
+    # with the rest of anago but replicate the pattern at your peril
+    # definitely understand what's happening when you do
+    #
+    # myVar="OG"; ((0)) && myVar="changed"; echo $myVar
+    #
+    # vs
+    #
+    # myVar="OG"; ((1)) && myVar="changed"; echo $myVar
+    # note also that the values for FLAGS_x are either 0 (not set)
+    # or 1 (set) so that this works. This is not Ruby or similar
+    # where "--some-flag" would be truthy but it is BASH
+    # so values > 0 are actually truthy
```

Comment:
In describing how bash exit codes work, this would be more useful as a top level .md file also highlighting some style elements of this repo?

Reply:
That's probably true, @david-mcmahon, but since we're using some expansion features of the shell so that something evaluating to `>0` returns an exit code of 0 it seemed as if linking to a few explainers was a reasonable first step